lib/string.gi

```diff
@@ -1406,3 +1406,61 @@ InstallGlobalFunction(PrintFormatted, function(args...)
     # directed
     Print(CallFuncList(StringFormatted, args));
 end);
+
+InstallGlobalFunction(Pluralize,
+function(args...)
+  local nargs, str, len;
+
+  nargs := Length(args);
+  if not (nargs in [2, 3] and
+          IsInt(args[1]) and args[1] >= 0 and
+          ForAll([2 .. nargs], i -> IsString(args[i]))) then
+    ErrorNoReturn("Usage: Pluralize(<count>, <string>[, <plural>])");
+  elif args[1] = 1 then
+    return args[2];
+  elif nargs = 3 then
+    return args[3];
+  fi;
+  str := args[2];
+  len := Length(str);
+
+  # Special case weirdly short words
+  if len <= 1 then
+    return str;
```

Comment:
perhaps it would better to raise an error here, stating that we don't know how to pluralize this word. Or else make a leap and just add "s". Then wit would for construct like "there are 5 Bs in the sentence".

But just returning `str` unchanged seems like a way to introduce subtle bugs. But perhaps I am overly critical :-)

Reply:
I've gone with len = 0 is an error, otherwise add an `s` for 1- and 2-letter words.